gnovm/pkg/gnolang/type_check_test.go

```diff
@@ -7,6 +7,7 @@ func TestCheckAssignableTo(t *testing.T) {
 
 	tests := []struct {
 		name       string
+		n          Node
```

Comment:
if this is always nil, just pass `nil` below

Reply:
Yes right my bad

Reply:
done here https://github.com/gnolang/gno/pull/3068/commits/7ae5d59fd3ba2303cc25c4e807a8ae6f685c617c